Uncomment a print statement, as instructed by the HISTORY file, to prevent a
crash on (some verisons of?) macOS.
--- docker/docker.f.orig	2023-05-26 03:31:37.000000000 -0500
+++ docker/docker.f	2023-08-30 04:54:37.000000000 -0500
@@ -1913,7 +1913,7 @@
 c print is to prevent segment violation on MacOSX gfortran
 c apple-darwin15. Uncomment next line
 c newer versions of grfortran may not have this bug
-c      print*," "
+      print*," "
       if (iflat.gt.0) then
           oflat = .true.
       else 
